src/DrawerLayout.js

```diff
@@ -212,7 +212,7 @@ export default class DrawerLayout extends Component {
   }
 
   _shouldSetPanResponder = (e, { moveX, dx, dy }) => {
-    if (!dx || !dy) {
+    if (!dx || !dy || Math.abs(dx) < 3) {
```

Comment:
It seemed to be a good balance between your finger pressing straight down causing dx to fluctuate slightly and not interfering with animations if you were swiping left and right.